specification/network/resource-manager/Microsoft.Network/stable/2024-01-01/examples/NetworkWatcherFlowLogUpdateTags.json

```diff
@@ -23,6 +23,7 @@
           "targetResourceId": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Network/networkSecurityGroups/desmondcentral-nsg",
           "targetResourceGuid": "00000000-0000-0000-0000-000000000000",
           "storageId": "/subscriptions/subid/resourceGroups/rg1/providers/Microsoft.Storage/storageAccounts/nwtest1mgvbfmqsigdxe",
+          "enabledFilteringCriteria": "srcIP=158.255.7.8 || dstPort=56891",
```

Comment:
>enabledFilteringCriteria [](http://example.com/codeflow?start=11&length=24)

Why is this needed here ?

Reply:
We want to return all the properties related to flowlogs resource in the update response too..